packages/ui-library/src/elements/toast/index.js

```diff
@@ -149,17 +155,20 @@ const Toast = ( {
 					"yst-toast",
 					className,
 				) }
-				role="alert"
+				role={ role }
+				ref={ ref }
+				{ ...props }
 			>
 				{ children }
 			</Transition>
 		</ToastContext.Provider>
 	);
-};
+} );
 
 Toast.propTypes = {
 	children: PropTypes.node,
 	id: PropTypes.string.isRequired,
+	role: PropTypes.string.isRequired,
```

Comment:
Does this really need to be a required prop breaking API change?

You can already add it via the prop spread now. So you're saying noone may use a Tooltip without specifying a `role`.
IMO this seems a bit overzealous for the purpose of an `element` in our UI library.